composer.json

```diff
@@ -15,17 +17,23 @@
         "source": "https://github.com/workivate/php-saml/"
     },
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.1",
```

Comment:
To me `>=` feels a risky, we can't really know if PHP9 won't introduce some breaking change making it to stop working, maybe let's make it `^8.1`?

Reply:
Yep, seems like a good approach. The upstream does it this way because that supports multiple versions of PHP, but I can't see why we need to stay with that, especially as the upstream is not in active development